_vector-search/vector-search-techniques/index.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: default
+title: Vector search techniques
+nav_order: 15
+has_children: true
+has_toc: false
+redirect_from:
+  - /search-plugins/knn/
+  - /search-plugins/knn/index/ 
+  - /vector-search/vector-search-techniques/     
+---
+
+# Vector search techniques
+
+OpenSearch implements vector search as *k-nearest neighbors*, or *k-NN*, search. k-NN search finds the k neighbors closest to a query point across an index of vectors. To determine the neighbors, you can specify the space (the distance function) you want to use to measure the distance between points.
+
+OpenSearch supports three different methods for obtaining the k-nearest neighbors from an index of vectors:
+
+- [Approximate search]({{site.url}}{{site.baseurl}}/search-plugins/knn/approximate-knn/)) (approximate k-NN, or ANN): Returns approximate nearest neighbors to the query vector. Usually, approximate search algorithms sacrifice indexing speed and search accuracy in exchange for performance benefits such as lower latency, smaller memory footprints, and more scalable search. For most use cases, approximate search is the best option.
+
+- Exact search: A brute-force, exact k-NN search of vector fields. OpenSearch supports the following types of exact search: 
+  - [Exact search with scoring script]({{site.url}}{{site.baseurl}}/search-plugins/knn/knn-score-script/): Using a scoring script, you can apply a filter to an index before executing the nearest neighbor search. 
+  - [Painless extensions]({{site.url}}{{site.baseurl}}/search-plugins/knn/painless-functions/): Adds the distance functions as Painless extensions that you can use in more complex combinations. You can use this method to perform a brute-force, exact vector search of an index, which also supports pre-filtering. 
+
+
+Overall, for larger data sets, you should generally choose the approximate nearest neighbor method because it scales significantly better. For smaller data sets, where you may want to apply a filter, you should choose the custom scoring approach. If you have a more complex use case where you need to use a distance function as part of their scoring method, you should use the Painless scripting approach.
+
+## Approximate search
+
+OpenSearch supports the following algorithms (_methods_) for approximate vector search:
```

Comment:
Can we shorten this section to just point to the engines/methods docs? We should just say that OpenSearch supports a variety of different backend engines and algorithms. Based on your mode and memory requirements, OpenSearch will select the best default configuration.